src/uct/ib/base/ib_iface.c

```diff
@@ -62,153 +62,171 @@ static const char *uct_ib_iface_addr_types[] = {
 };
 
 ucs_config_field_t uct_ib_iface_config_table[] = {
-  {"", "", NULL,
-   ucs_offsetof(uct_ib_iface_config_t, super), UCS_CONFIG_TYPE_TABLE(uct_iface_config_table)},
-
-  {"SEG_SIZE", "8192",
-   "Size of bounce buffers used for post_send and post_recv.",
-   ucs_offsetof(uct_ib_iface_config_t, seg_size), UCS_CONFIG_TYPE_MEMUNITS},
-
-  {"TX_QUEUE_LEN", "256",
-   "Length of send queue in the QP.",
-   ucs_offsetof(uct_ib_iface_config_t, tx.queue_len), UCS_CONFIG_TYPE_UINT},
-
-  {"TX_MAX_BATCH", "16",
-   "Number of send WQEs to batch in one post-send list. Larger values reduce\n"
-   "the CPU usage, but increase the latency and pipelining between sender and\n"
-   "receiver.",
-   ucs_offsetof(uct_ib_iface_config_t, tx.max_batch), UCS_CONFIG_TYPE_UINT},
-
-  {"TX_MAX_POLL", "16",
-   "Max number of receive completions to pick during TX poll",
-   ucs_offsetof(uct_ib_iface_config_t, tx.max_poll), UCS_CONFIG_TYPE_UINT},
-
-  {"TX_MIN_INLINE", "64",
-   "Bytes to reserve in send WQE for inline data. Messages which are small\n"
-   "enough will be sent inline.",
-   ucs_offsetof(uct_ib_iface_config_t, tx.min_inline), UCS_CONFIG_TYPE_MEMUNITS},
-
-  {"TX_INLINE_RESP", "0",
-   "Bytes to reserve in send WQE for inline response. Responses which are small\n"
-   "enough, such as of atomic operations and small reads, will be received inline.",
-   ucs_offsetof(uct_ib_iface_config_t, inl[UCT_IB_DIR_TX]), UCS_CONFIG_TYPE_MEMUNITS},
-
-  {"TX_MIN_SGE", "4",
-   "Number of SG entries to reserve in the send WQE.",
-   ucs_offsetof(uct_ib_iface_config_t, tx.min_sge), UCS_CONFIG_TYPE_UINT},
+    {"", "", NULL, ucs_offsetof(uct_ib_iface_config_t, super),
```

Comment:
pls revert the styling change of uct_ib_iface_config_table, we usually do not make large code changes in existing code to satisfy clang-format rules